homeassistant/components/media_player/spotify.py

```diff
@@ -20,9 +22,8 @@
     CONF_NAME, STATE_PLAYING, STATE_PAUSED, STATE_IDLE, STATE_UNKNOWN)
 import homeassistant.helpers.config_validation as cv
 
-COMMIT = '544614f4b1d508201d363e84e871f86c90aa26b2'
-REQUIREMENTS = ['https://github.com/happyleavesaoc/spotipy/'
-                'archive/%s.zip#spotipy==2.4.4' % COMMIT]
+REQUIREMENTS = ['https://github.com/plamere/spotipy/'
+                'archive/master.zip#spotipy==2.4.4']
```

Comment:
Just checked and spotipy is available in pypi in version 2.4.4

No need for tor this hack...

Reply:
It is needed, because the current PyPi 2.4.4 package does not include this PR: https://github.com/plamere/spotipy/pull/182

The pypi package is from January 2017.
At least I wanted to upgrade from @happyleavesaoc repo to the original master repo.

Reply:
Just download the code from pypi and GitHub 

This package is 💩... same version and different content :(

Reply:
Yeah, definitely track the `spotipy` release instead of my fork. w.r.t. pypi version, issue has been open for awhile: https://github.com/plamere/spotipy/issues/211

Reply:
Maybe contacting @plamere directly through email ?

Reply:
Should this really a blocking point for that PR? I mean it was a GitHub repo before and it is a Github repo after.

Reply:
I don't have the power to merge :)

But the guidelines are well defined: only pypi packages in requirements

Reply:
That means I cannot improve a component which is using a GitHub repo since years? Don't touch that component otherwise you are forced tp persuade the pypi owner to update the package or maintain your own package for us.

Reply:
The reasoning is that the master.zip can change and break stuff, while pypi packages will perdure in time.

Reply:
Great - Then I will revert to @happyleavesaoc commit .zip 👍

Reply:
I will leave it as it is for now - If someone wants to merge it, he can just revert the 618e7510302ac218877cdfc489532f3fc7b19421